desktop/static/setup.tsx

```diff
@@ -33,6 +34,7 @@ export type Config = {
   updaterEnabled?: boolean;
```

Comment:
Curious, what does this config setting do then?

Reply:
FWIW, I never found any usages of this in the code...initially thought this was used to disable the updater when I was trying to see if there was an undocumented feature...